src/libraries/System.Private.CoreLib/src/System/Guid.cs

```diff
@@ -1085,7 +1093,7 @@ public int CompareTo(Guid value)
             if (value._k != _k)
             {
                 return GetResult(_k, value._k);
-            }
+                }
```

Comment:
Nit: indentation increased
```suggestion
            }
```

Reply:
Thanks, I've adjusted my show leading/trailing whitespace differences setting.